setup/templates/steps/config.html

```diff
@@ -58,13 +58,6 @@
   </p>
 </div>
 
-<div class="form-check form-check-inline">
-  <label class="form-check-label">
-    <input class="form-check-input" type="checkbox" name="disable_statistics" value="True">
-    Opt-out of statistics
```

Comment:
Hmm, I wouldn't remove this without removing the code in admin. Can't we fix it instead?

Reply:
It has never worked. It requires a separate webserver to run (https://github.com/Mailu/Stats). This webserver has never been deployed.  

But I agree, if this is removed then the code in admin should be removed as well. I didn't think about admin.

Reply:
I could reinstate the code, but will simply hide the element and set it to disabled in mailu.env. 
That way we can still choose if we want to implement it in the future by hosting the webserver required for this.